sunpy/map/sources/soho.py

```diff
@@ -153,6 +153,32 @@ def is_datasource_for(cls, data, header, **kwargs):
         """Determines if header corresponds to an LASCO image."""
         return header.get('instrume') == 'LASCO'
 
+    @property
+    def mask_occulter(self):
+        # Returns a mask for the LASCO map
+        from astropy.io import fits
+
+        from sunpy.map.maputils import all_coordinates_from_map
+
+        x_occ = self.meta['crval1']/self.meta['cdelt1']+self.meta['crpix1']
+        y_occ = self.meta['crval2']/self.meta['cdelt2']+self.meta['crpix2']
+        occ_coord = self.pixel_to_world(x_occ*u.pixel, y_occ*u.pixel)
+        hpc_coords = all_coordinates_from_map(self)
+        norm_mask = np.sqrt((hpc_coords.Tx-occ_coord.Tx)**2 +
+                            (hpc_coords.Ty-occ_coord.Ty)**2)
+        if self.detector == 'C2':
+            r_in = 2.4
+            r_out = 6.0
+        if self.detector == 'C3':
+            c3mask = fits.open("https://hesperia.gsfc.nasa.gov/ssw/soho/lasco/idl/data/calib/c3_cl_mask_lvl1.fts")
```

Comment:
This scares me :rofl:

I think that the best thing to do for this data file is use [the data manager](https://docs.sunpy.org/en/latest/dev_guide/contents/remote_data.html).